easy_my_coop_website/views/subscription_template.xml

```diff
@@ -89,16 +89,20 @@
                                 class="form-horizontal mt32"
                                 enctype="multipart/form-data"
                             >
-                                <p style="color:red;">
-                                    <t t-esc="error_msg" />
-                                </p>
-                                <input
-                                    type="hidden"
-                                    name="csrf_token"
-                                    t-att-value="request.csrf_token()"
-                                />
-
-                                <div t-attf-class="form-group">
+                                <div name="crsf_token_container">
+                                    <p style="color:red;">
+                                        <t t-esc="error_msg" />
+                                    </p>
```

Comment:
This banner is used to display error message that may be unrelated to csrf tokens. It should be out of the csrf div.